tests/test_mg_convert/test_mg_convert.py

```diff
@@ -154,6 +154,11 @@ def _run_openmc(self):
             form = '{0:12.6E} {1:12.6E}\n'
             outstr += form.format(sp.k_combined[0], sp.k_combined[1])
 
+            # Enforce closing statepoint and summary files so HDF5
+            # does not throw an error during the next OpenMC execution
+            sp._f.close()
+            sp._summary._f.close()
```

Comment:
What error are you seeing without these lines? I don't see any errors without, so I'm just curious what difference is. What version of HDF5/h5py are you using?